frontend/src/pages/order-status/libs/components/buttons-section/buttons-section.tsx

```diff
@@ -24,6 +33,26 @@ const ButtonsSection = ({
     status === OrderStatus.DONE ||
     status === OrderStatus.REJECTED;
 
+  const stripeDataStatus = useAppSelector(selectStripeDataStatus);
+  const isFetching =
+    stripeDataStatus === DataStatus.PENDING ||
+    stripeDataStatus === DataStatus.FULFILLED;
```

Comment:
I'm not sure why Fulfilled means that data is fetching. 🤔

Reply:
Because othevise the button becomes active for a brief moment until stripe catches redirect, and it will be possible to press it for the second time

Reply:
but you right, i should rename it